web/controllers/page_controller.ex

```diff
@@ -1,16 +1,20 @@
 defmodule Askbywho.PageController do
+  require Logger
   @moduledoc """
    Add documentation
   """
 
   use Askbywho.Web, :controller
   alias Askbywho.Email
+  alias Askbywho.Brand
 
   plug :put_layout, "site.html"
 
   def index(conn, _params) do
+    brands = Repo.all(Brand)
+    name_brands = brands |> Enum.map(&{&1.name, &1.name})
```

Comment:
Is there a reason you need a tuple of the same name?  Wouldn't this `Enum.map(&(&1.name))` be sufficient to return a list of brand names?